docs/3.x.x/api-reference/reference.md

```diff
@@ -27,6 +28,11 @@
 
 This object contains the controllers, models, services and configurations contained in the `./admin` folder.
 
+## strapi.api
+
+This object contains APIs contained in the `./api` folder.
```

Comment:
This object contains the models from the `./api` folder

Reply:
There is no `models` here...?

Reply:
I was trying to find a synonym for API since what you mean by api is model I thought it was appropriate